roles/jenkins_job_launcher/README.md

```diff
@@ -0,0 +1,26 @@
+# jenkins_job_launcher
+
+jenkins_job_launcher role triggers Jenkins job using the uri module.
+the role makes use of username + user API token to authenticate with Jenkins.
+
+Role Variables
+--------------
+
+Required variables are: 
+job_url - Jenkins job url
+job_params - job params and values based on the following playbook example.
+username - Jenkins user
+token - the user API token
```

Comment:
We don't enforce [role variable naming through linting check](https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/.ansible-lint#L15) but we do require all new roles to follow some sort of role variable naming. For example, role variable linting expects variables with the prefix `jenkins_job_launcher_` but we understand it could be quite long, in those cases using something like `jjl_` is good enough.

So `job_url` becomes `jjl_job_url` or perhaps `jjl_url`, etc.